### PR TITLE
Fix e2e and wc

### PR DESCRIPTION
**Pull request by brunobar79**
- Fixes broken e2e
- Remove tiny badge from WC prompt
- Fixes wrong wallet balance in WC tx request for L2s
- Fixes wrong native asset calculations in WC tx request
- Fix flash /  jumping on WC requests

PoW: https://recordit.co/1kkGM8YAoE

### CODE REVIEW

File: src/utils/ethereumUtils.js [hunk 1]
```diff
@@ -67,7 +68,40 @@ const getNativeAssetForNetwork = (allAssets, network) => {
     default:
       nativeAssetAddress = ETH_ADDRESS;
   }
-  return getAsset(allAssets, toLower(nativeAssetAddress));
+  const { accountAddress } = store.getState().settings;
+  let differentWallet = toLower(address) !== toLower(accountAddress);
+  const { assets } = store.getState().data;
+  let nativeAsset =
```
Comment (estebanmino) [inline src/utils/ethereumUtils.js:74]: is `let` needed?

**Review by estebanmino — APPROVED** [review]
code lgtm!

File: src/utils/ethereumUtils.js (hunk 1 above)
Comment (brunobar79) [reply src/utils/ethereumUtils.js:74]: Yes!

**Review by skylarbarrera — APPROVED** [review]
🤝

**Pull request merged**